qiskit_nature/second_q/algorithms/excited_states_solvers/qeom.py

```diff
@@ -275,19 +354,27 @@ def solve(
 
         # 1. Prepare all operators and set the particle number in the qubit converter
         (
-            untap_main_op,  # Hamiltonian
-            untap_aux_ops,  # Auxiliary observables
+            untap_main_op_sumop,  # Hamiltonian
+            untap_aux_ops_sumop,  # Auxiliary observables
         ) = self.get_qubit_operators(problem, aux_operators)
 
+        untap_main_op = untap_main_op_sumop.primitive
+        untap_aux_ops = {key: op.primitive for key, op in untap_aux_ops_sumop.items()}
+
         # 2. Run ground state calculation with fully tapered custom auxiliary operators
         # Note that the solve() method includes the `second_q' auxiliary operators
         if isinstance(self.qubit_converter, QubitConverter):
-            tap_aux_operators = self.qubit_converter.symmetry_reduce_clifford(untap_aux_ops)
+            tap_aux_operators_sumop = self.qubit_converter.symmetry_reduce_clifford(
+                untap_aux_ops_sumop
```

Comment:
Given the above, why do you continue to use the sumop variants here?

Reply:
All public method have a PauliSumOp interface. I changed some notations to make it appear clearly. In this case, `get_qubit_operators()` used to return `untapped_aux_ops` but I renamed it to `untapped_aux_ops_sumop` to make the typing clear. The public method `symmetry_reduce_clifford()` also has a PauliSumOp interface